### PR TITLE
fix: video access update

### PR DESCRIPTION
**Pull request by JohaneDecamps**
fix : route update video 
add: video update component

### CODE REVIEW

File: client/src/components/videoforms/VideoUpdate.jsx [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+  } = useForm();
+
+  const onSubmit = async (data) => {
+    try {
+      axios
+        .put(`${expressURL}/api/videos/${data.id}`, data)
+        toast.success("Video access updated successfully ! ");
+    } catch (err) {
+
+      toast.err("An error occured, please try again later");
```
Comment (aurelienLavanchy) [inline client/src/components/videoforms/VideoUpdate.jsx:22]: `.err` is not a toastify method, use `.error` for error messages instead

**Review by ioayoub — APPROVED** [review]

File: client/src/components/videoforms/VideoUpdate.jsx [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+
+  useEffect(() => {
+    const express = import.meta.env.VITE_API_URL;
+    try {
+      axios.get(`${express}/api/videos`).then((response) => {
+        const {data} = response;
+        setVideoData(data);
+      });
+    } catch (err) {
+      console.error(err);
```
Comment (aurelienLavanchy) [inline client/src/components/videoforms/VideoUpdate.jsx:35]: replace the `console.error` by an error toast

File: client/src/components/videoforms/VideoUpdate.jsx [hunk 3]
```diff
@@ -0,0 +1,65 @@
...
+          </option>
+        ))}
+      </select>
+</div>
+      <input type="radio" name="free" value="0" {...register("is_connected")} />
+
+      <label htmlFor="acces"> Free </label>
+
+      <input
+        type="radio"
```
Comment (aurelienLavanchy) [inline client/src/components/videoforms/VideoUpdate.jsx:56]: a checkbox might be more intuitive than a radio button

File: client/src/components/videoforms/VideoUpdate.jsx [hunk 4]
```diff
@@ -0,0 +1,65 @@
...
+      <label htmlFor="videoUpdate"> Access </label>
+      <select name="videoupdate" {...register("id")}>
+        {videoData?.map((v) => (
+          <option key={v.id} value={v.id}>
+            {`${v.title.charAt(0).toUpperCase()}${v.title.slice(1)}`}
+          </option>
+        ))}
+      </select>
+</div>
+      <input type="radio" name="free" value="0" {...register("is_connected")} />
```
Comment (aurelienLavanchy) [inline client/src/components/videoforms/VideoUpdate.jsx:51]: same as below for booleans, use radio buttons to select between different options instead

**Review by MichaelLELU — APPROVED** [review]

**Review by aurelienLavanchy — APPROVED** [review]

**Review by camille-haxa — APPROVED** [review]

**Review by ioayoub — APPROVED** [review]

**Pull request merged**